### PR TITLE
Search tidy up

### PR DESCRIPTION
**Pull request by Gizem03**
fixed the look of cards a bit, added date to verify date filter and various other fixes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
     <nav class="navbar navbar-expand-lg navbar-light bg-light w-50">
       <div class="container-fluid">
-        <a class="navbar-brand" href="#">Navbar</a>
+        <a class="navbar-brand" href="#">Event Finder</a>
```
[the remark this answers is not in the document]
Comment (Gizem03) [reply index.html:21]: cool, i'll add it

File: search.html [hunk 2]
```diff
@@ -26,10 +26,9 @@
     ></script>
   </head>
   <body>
-    
     <nav class="navbar navbar-expand-lg navbar-light bg-light">
       <div class="container-fluid">
-        <a class="navbar-brand" href="#">Local Event App</a>
+        <a class="navbar-brand" href="#">Event Finder</a>
```
Comment (MichalMatu) [inline search.html]: href search.html

File: search.html [hunk 3]
```diff
@@ -119,7 +116,5 @@ <h4>Filters</h4>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.4/moment.min.js"></script>
     <script src="asset/search.js"></script>
```
Comment (MichalMatu) [inline search.html:117]: duplicated

**Review by MichalMatu — CHANGES_REQUESTED** [review]
some changes nedeed

File: search.html (hunk 2 above)
Comment (Gizem03) [reply search.html]: added but as index.html as the title of the app should direct to index page

File: search.html (hunk 3 above)
Comment (Gizem03) [reply search.html:117]: what is duplicated here?

**Review by MichalMatu — APPROVED** [review]

**Pull request merged**